custom_components/healthchecksio/sensor.py

```
@@ -6,95 +6,99 @@
+    async_add_devices(
+        HealthchecksioSensor(
+            hass=hass,
+            ping_url=check.get(ATTR_PING_URL),
+            coordinator=coordinator,
+            config_entry=config_entry,
+        )
+        for check in coordinator.data.get("checks", [])
+    )
 
 
 class HealthchecksioSensor(SensorEntity):
```

Pylint (3.3.7)

[refactor] Too many instance attributes (10/7)

(R0902)